apps/demo-app/src/app/examples/search-filter/search-filter/users.ts

```diff
@@ -32,7 +32,7 @@ enum Country {
     southAfrica = 'South Africa'
 }
 
-class User {
+interface User {
```

Comment:
Is this changed related to the angular upgrade, and if yes, how?

Reply:
not directly related to angular update

Reply:
removing this nonsense below 
```diff
-
-    constructor(data?: Partial<User>) {
-        if (data) {
-            for (const property in data) {
-                if (data.hasOwnProperty(property)) { (this)[property] = (data)[property]; }
-            }
-        }
-    }
```